src/blocks/notify.rs

```diff
@@ -11,14 +9,16 @@
 //! `driver` | Which notifications daemon is running. Available drivers are: `"dunst"` and `"swaync"` | `"dunst"`
 //! `format` | A string to customise the output of this block. See below for available placeholders. | `" $icon "`
 //!
-//! Placeholder | Value                                      | Type   | Unit
-//! ------------|--------------------------------------------|--------|-----
-//! `icon`      | Icon based on notification's state         | Icon   | -
-//! `paused`    | Present only if notifications are disabled | Flag   | -
+//! Placeholder          | Value                                      | Type   | Unit
+//! ---------------------|--------------------------------------------|--------|-----
+//! `icon`               | Icon based on notification's state         | Icon   | -
+//! `notification_count` | The number of notification (omitted if 0)  | Number | -
+//! `paused`             | Present only if notifications are disabled | Flag   | -
 //!
 //! Action          | Default button
 //! ----------------|---------------
 //! `toggle_paused` | Left
+//! `show`          | -
```

Comment:
An example of how to map this action might be useful, since it's a new feature.

Reply:
Added an example :)